internal/file/filesystem.go

```diff
@@ -35,7 +35,7 @@ func NewLocalFS() (Filesystem, error) {
 	fs := filesystem.DefaultFs{}
 
 	for _, directory := range directories {
-		err := fs.MkdirAll(directory, 0655)
+		err := fs.MkdirAll(directory, 0777)
```

Comment:
Just curious about the kind of problem you've been facing using 0655?

Reply:
I get permission errors

Reply:
Not sure if I get it, if both the controller and NGINX use the same `www-data` user, where do permission errors come from?

Reply:
Thank you for keep asking about this. I just added a commit that removes this numeric values with constants. I will find out why this is failing in the e2e tests.